pkg/codegen/pcl/invoke.go

```diff
@@ -44,6 +44,55 @@ func getInvokeToken(call *hclsyntax.FunctionCallExpr) (string, hcl.Range, bool)
 	return literal.Val.AsString(), call.Args[0].Range(), true
 }
 
+// annotateObjectProperties annotates the properties of an object expression with the types of the corresponding
+// properties in the schema. This is used to provide type information
+// for invoke calls that didn't have type annotations.
+//
+// this function will recursively annotate the properties of objects
```

Comment:
Nit: capitalization consistency
```suggestion
// This function will recursively annotate the properties of objects
```